tools/eksDistroBuildToolingOpsTools/docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-builder:latest
+
+ARG BINARY_OUTPUT_PATH
+
+ADD $BINARY_OUTPUT_PATH /usr/bin
+
+RUN chmod +x /usr/bin/eksDistroOpsProwPlugin
```

Comment:
let's parameterize `eksDistroOpsProwPlugin` somehow, wether it's through another ARG or w/e, so that we don't have to change this value if we cahnge the binary name, or if we want to re-use this dockerfile for building other things